From 75205d25b405623a0b2f0a3446fdd15fba9ac6d5 Mon Sep 17 00:00:00 2001
From: Mihails Strasuns <mihails.strasuns.contractor@sociomantic.com>
Date: Tue, 9 Feb 2016 20:37:37 +0200
Subject: [PATCH] Use sc-transitional.ini/dmd-transitional.conf

Contrary to 53aea6baccd4ce7fa6db62afe3788604b9484614 it doesn't
look for plain dmd.conf as fallback
---
 src/mars.d | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mars.d b/src/mars.d
index 22e1bcbf7..d5d728154 100644
--- a/src/mars.d
+++ b/src/mars.d
@@ -384,11 +384,11 @@ extern (C++) int tryMain(size_t argc, const(char)** argv)
     {
         version (Windows)
         {
-            global.inifilename = findConfFile(global.params.argv0, "sc.ini");
+            global.inifilename = findConfFile(global.params.argv0, "sc-transitional.ini");
         }
         else static if (__linux__ || __APPLE__ || __FreeBSD__ || __OpenBSD__ || __sun)
         {
-            global.inifilename = findConfFile(global.params.argv0, "dmd.conf");
+            global.inifilename = findConfFile(global.params.argv0, "dmd-transitional.conf");
         }
         else
         {
-- 
2.11.0

